tests/test_parser.py

```diff
@@ -74,3 +74,41 @@ def test_function_arguments_validation(self):
 
         with self.assertRaises(ParseError):
             parse_one("IF(a > 0)")
+
+    def test_cache_table(self):
```

Comment:
move to test_dialects

Reply:
I put it in identity.sql as well, since I just want to verify parsing works and UNCACHE / CACHE is a Spark concept only.